src/components/LandingComp/FAQSection.tsx

```diff
@@ -103,10 +103,10 @@ const FAQSection = () => {
       question: "What data format does Chords support?",
       answer: (
         <>
-          Chords supports an array format: [counter, A0, A1, ..., A4], where
-          counter is a uint8_t (0-255) and A0-A4 are raw signal values. Array
-          example : [10, 468, 472, 463, 466, 465]. For implementation details,
-          see our{" "}
+          Chords supports an array format: [A0, A1, ..., A5, counter], where
+           A0-A5 are raw signal values and counter is a uint8_t (0-255). Array
+          example : [468, 472, 463, 466, 465, 434, 10]. For implementation
+          details, see our{" "}
           <Link
             href="https://github.com/upsidedownlabs/Chords-Web/blob/main/ArduinoFirmware/ArduinoFirmware.ino"
             target="_blank"
```

Comment:
Change this to point to our new Arduino firmware repository.

Reply:
I will fix it in next pull request

Reply:
I have fixed this issue in [PR #52 of the Chords-Web repository](https://github.com/upsidedownlabs/Chords-Web/pull/52).